data/features/mmex.json

```diff
@@ -4,8 +4,8 @@
     "features": [
         {  "name":"accounts",
             "icon": "fas fa-money-check",
-            "title": "Accounts & Currencies",
-            "text": "Wizard to simply create accounts and start to use MMEX. Multiple currencies for each account to have more flexibility."
+            "title": "Accounts and currencies",
```

Comment:
Both "Accounts & Currencies" and "Accounts and currencies" are grammatically correct, but they convey slightly different tones and styles.

"Accounts & Currencies" is more concise and often used in headings, titles, or menus where brevity is preferred. The use of the ampersand "&" can give a more streamlined and professional appearance, and it's commonly seen in business contexts.

"Accounts and currencies" is slightly more formal and may be preferred in more detailed or explanatory contexts where clarity is essential. The use of "and" spells out the conjunction fully, which can be helpful for ensuring understanding, especially if the text is longer or more complex.

Ultimately, the choice between them depends on the context and the desired tone or style of your communication. If you're aiming for brevity and a modern appearance, "Accounts & Currencies" might be more suitable. If you prefer a more formal or explanatory tone, "Accounts and currencies" could be a better choice.

Reply:
Using sentence case consistent with:
* https://www.google.com/docs/about/#features
* https://www.skype.com/en/features/

It seems that when words which start with a capital letter, they are proper nouns.